### PR TITLE
Glass convert

### PR DESCRIPTION
**Pull request by ori-steinberg**
Minor modification to the Glass model for compatibility with TorchScript conversion

### CODE REVIEW

File: glass/inference/glass_runner.py [hunk 1]
```diff
@@ -1,42 +1,35 @@
...
-from detectron2.structures import Instances
-
-from glass.config import add_e2e_config, add_glass_config, add_dataset_config, add_post_process_config
-from glass.postprocess import build_post_processor
 from glass.utils.common_utils import rgb2grey
+from glass.postprocess import build_post_processor
+from detectron2.checkpoint import DetectionCheckpointer
+from detectron2.structures import Instances, RotatedBoxes
 from glass.modeling.recognition.text_encoder import TextEncoder
+from Trax.Miscellaneous.Users.OriS.OCR.glass_det.glass_main.glass.utils.visualizer import visualize
```
Comment (idonahum1) [inline glass/inference/glass_runner.py]: Are you sure you want to import something for personal folder? maybe you should move it Trax.Algo.External under name GlassOCR

File: glass/inference/glass_runner.py [hunk 2]
```diff
@@ -1,42 +1,35 @@
...
         Initializes a runner that can run inference using a detectron2 based model
-        :param model_path: Path to the detectron2 model
-        :param config_path: Path to the configuration file of the detectron2 model
-        :param opts: Additional option pairs to override settings in the configuration
-        :param post_process: Whether to run post-processing or not
+        Args:
+            model_path: Path to the detectron2 model
+            config_path: Path to the configuration file of the detectron2 model
+            opts: Additional option pairs to override settings in the configuration
+            post_process: Whether to run post-processing or not
```
Comment (idonahum1) [inline glass/inference/glass_runner.py:28]: I think we use sphinx style to do doc string in the garage. It doesnt really matter for me, you choice if you want to change it.

File: glass/modeling/roi_heads/rotated_fast_rcnn.py [hunk 3]
```diff
@@ -363,14 +362,13 @@ def inference(self, score_thresh, nms_thresh, topk_per_image):
                 oriented_box = overwrite_orientations_on_boxes(box, orientation_values)
                 oriented_boxes.append(oriented_box)
             boxes = tuple(oriented_boxes)
-        else:
-            boxes = [boxes]
+        # else:
+        #     boxes = [boxes]
```
Comment (idonahum1) [inline glass/modeling/roi_heads/rotated_fast_rcnn.py]: If this comment is not needed just delete it.

File: glass/inference/glass_runner.py (hunk 1 above)
Comment (ori-steinberg) [reply glass/inference/glass_runner.py]: Good catch, I thought I already removed it.

File: glass/inference/glass_runner.py (hunk 2 above)
Comment (ori-steinberg) [reply glass/inference/glass_runner.py:28]: I prefer this style, it is shorter, clearer, and more elegant

**Review by idonahum1 — APPROVED** [review]

**Pull request merged**